src/js/models/v2/state/BaseStateProfile.js

```diff
@@ -49,7 +50,7 @@ const BaseStateProfile = {
     },
     get flag() {
         if (this.id) {
-            return `https://s3-us-gov-west-1.amazonaws.com/da-public-files/usaspending_state_flags/${this.id}.png`;
+            return `https://files${kGlobalConstants.DEV ? '-nonprod' : ''}.usaspending.gov/state-flags/${this.id}.png`;
```

Comment:
The flag image links in under "State Profile" appear broken. Please change the  "-" in "state-flags" to "_"